model/project_selector.go

```diff
@@ -275,6 +275,38 @@ func (t *taskSelectorEvaluator) evalSelector(s Selector) ([]string, error) {
 	return results, nil
 }
 
+// Display task selector
+type displayTaskSelectorEvaluator struct {
+	tagEval *tagSelectorEvaluator
+}
+
+func NewDisplayTaskSelectorEvaluator(bv BuildVariant, tasks []parserTask) *displayTaskSelectorEvaluator {
+	var selectees []tagged
+	for _, t := range bv.Tasks {
+		selectees = append(selectees, &parserTask{Name: t.Name, Tags: getTags(tasks, t.Name)})
+	}
+	return &displayTaskSelectorEvaluator{
+		tagEval: newTagSelectorEvaluator(selectees),
+	}
+}
+
+func getTags(tasks []parserTask, taskName string) parserStringSlice {
+	for _, t := range tasks {
+		if t.name() == taskName {
+			return t.tags()
+		}
+	}
+	return nil
+}
+
+func (t *displayTaskSelectorEvaluator) evalSelector(s Selector) ([]string, error) {
+	results, err := t.tagEval.evalSelector(s)
+	if err != nil {
+		return nil, errors.Wrap(err, "evaluating display task selector")
+	}
+	return results, nil
```

Comment:
aside from the wrapping, it looks like this is just a pass through?

Reply:
yep, I removed this